rs/bitcoin/ckbtc/minter/BUILD.bazel

```diff
@@ -109,7 +109,7 @@ rust_test(
     crate = ":ckbtc_minter_lib",
     deps = [
         # Keep sorted.
-        "@crate_index//:bitcoin",
+        "@crate_index//:bitcoin_0_28",
```

Comment:
understanding question: is there a reason why the code of the minter should not be upgraded?

Reply:
It is likely because there is another dependency `ic-btc-interface` is still on bitcoin 0.28. The dependency is from https://github.com/dfinity/bitcoin-canister, which was upgraded to bitcoin 0.32 just 3 days ago, and hasn't been made into a crates.io release yet.

Reply:
Or maybe I was wrong? Need further testing to figure this out. Anyway such an upgrade would break many tests, and I can give it a try in a another PR after this is merged.

Reply:
Thanks @ninegua for the explanation. I opened [XC-263](https://dfinity.atlassian.net/browse/XC-263) to track this and tentatively assigned it to you.